pom.xml

```diff
@@ -714,6 +714,10 @@ limitations under the License.]]></inlineHeader>
           <doclint>all,-missing</doclint>
           <excludePackageNames>com.datastax.*.driver.internal*</excludePackageNames>
           <tags>
+            <tag>
+              <name>apiNote</name>
+              <placement>A</placement>
+            </tag>
```

Comment:
This is supposed to be a "semi-official" tag in Java 8, IDEA recognizes it but the build doesn't.

Reply:
This tag is not being rendered in generated javadocs, is that on purpose?

Reply:
No, I thought it would but I didn't check. I might switch back to a regular paragraph.

Reply:
OK, it just needed an additional `<head>` element.